js/common/categories-results.js

```diff
@@ -0,0 +1,55 @@
+window.onload = () => {
+  const data = new URLSearchParams(window.location.search);
+
+  let ctype = data.get('c');
+
+  retrieveTitle(ctype);
+  retrieveProducts(ctype);
+  
+}
+
```

Comment:
Please move AJAX requests to this folder's db-methods folder. Also, create a separate file for insert, retrieve, update, and delete and name it that as well (ex: insert.js).

Reply:
Please see partner shop profile, be a partner/partner registration, or products page and their respective JavaScript files for reference.